__tests__/pages/exercises/[lessonSlug].test.js

```diff
@@ -64,10 +64,11 @@ describe('Exercises page', () => {
     // Previous button is not in the document on the first exercise.
     expect(queryByRole('button', { name: 'PREVIOUS' })).not.toBeInTheDocument()
 
-    const skipButton = getByRole('button', { name: 'SKIP' })
+    let skipButton = getByRole('button', { name: 'SKIP' })
     fireEvent.click(skipButton)
     expect(queryByRole('button', { name: 'PREVIOUS' })).toBeInTheDocument()
 
+    skipButton = getByRole('button', { name: 'SKIP' })
```

Comment:
The reason why we need to reassign `skipButton` to the current skip button in the DOM is because we are using a React key prop which makes it so the Exercise component rerenders.